File: 프로그래머스/unrated/181912. 배열 만들기 5/배열 만들기 5.js
function solution(intStrs, k, s, l) {
    const answer = [];
    
    intStrs.forEach(intStr => {
        if (intStr.slice(s, s + l) * 1 > k) {
            answer.push(intStr.slice(s, s + l) * 1);
        }
    });
    
    return answer;
}

File: 프로그래머스/unrated/181912. 배열 만들기 5/배열 만들기 5.java
import java.util.ArrayList;
import java.util.List;

class Solution {
    public int[] solution(String[] intStrs, int k, int s, int l) {
        List<Integer> result = new ArrayList<>();
        
        for (String str: intStrs) {
            int subInt = Integer.parseInt(str.substring(s, s + l));
            if (subInt > k) {
                result.add(subInt);
            }
        }
        
        return result.stream().mapToInt(i -> i).toArray();
    }
}